tests/reorder_python_imports_test.py

```diff
@@ -891,6 +891,18 @@ def test_separate_relative_and_separate_from():
     )
 
 
+def test_separate_relative_when_only_relative_imports_are_present():
+    ret = fix_file_contents(
+        'from . import foo\n'
+        'from . import bar\n',
+        separate_relative=True,
+    )
+    assert ret == (
+        'from . import bar\n'
+        'from . import foo\n'
+    )
```

Comment:
I would maybe write this as:

```python
src = ...
assert fix_file_contents(src, separate_relative=True) == src
```

so it's more clear that no change is expected

Reply:
sure. I just force pushed